charts/tekton-pipelines/values.yaml

```diff
@@ -229,6 +229,17 @@ buildpacks:
             chown -R "$(params.user_id):$(params.group_id)" "$(resources.inputs.app.path)"
             echo "semantic version updated"
 
+        - name: remove-dcproj-files
+          workingDir: $(resources.inputs.app.path)
+          image: "docker.io/library/bash:latest"
```

Comment:
Pls use version defined in the values yaml

```
# Default values for tekton-pipelines.
# This is a YAML-formatted file.
# Declare variables to be passed into your templates.

# -- default images used in our solution
# @default -- See below
images:
  # -- argocd cli downdload URL
  argocd_cli: https://github.com/argoproj/argo-cd/releases/download/v2.3.4/argocd-linux-amd64
  # -- bash image (used for various ops in steps)
  bash:      docker.io/library/bash:5.1.8
  # -- aws cli image (used for aws ecr auth)
  awscli:    docker.io/amazon/aws-cli:2.7.4
  # -- kaniko image used to build containers containing docker files - v1.8.1, uploaded April 5 2022
  kaniko:    gcr.io/kaniko-project/executor@sha256:b44b0744b450e731b5a5213058792cd8d3a6a14c119cf6b1f143704f22a7c650
  # -- git image
  git:       alpine/git:v2.32.0
  # -- kustomize cli
  kustomize: k8s.gcr.io/kustomize/kustomize:v4.5.4 # https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-kustomize/images.yaml
  # -- kubectl cli
  kubectl:   bitnami/kubectl:1.22.10
  # -- slack notifier
  slack:     cloudposse/slack-notifier:0.4.0
  # -- python image
  python:    saritasallc/python3:0.4 
```

Reply:
i.e `image: docker.io/library/bash:5.1.8`

Reply:
done